src/components/dashboard/Orders.tsx

```diff
@@ -1,93 +1,282 @@
 import * as React from "react";
-import Link from "@mui/material/Link";
-import Table from "@mui/material/Table";
-import TableBody from "@mui/material/TableBody";
-import TableCell from "@mui/material/TableCell";
-import TableHead from "@mui/material/TableHead";
-import TableRow from "@mui/material/TableRow";
-import Title from "./Title";
+import {Button } from "@material-ui/core";
+import { DataGrid, GridApi, GridKeyValue, GridColDef, GridValueGetterParams,GridRenderCellParams } from "@mui/x-data-grid";
+import Chip, { ChipProps } from "@material-ui/core/Chip";
+import { red, green } from "@material-ui/core/colors";
+import "../style/styling.css";
+import Typography from "@mui/material/Typography";
 
+/**
+ * paidChipProps is used to render a green or red chip around the paid column
+ * @param params Takes in the params of the GridRenderCellParams
+ * @returns returns the ChipProps
+ */
+function paidChipProps(params: GridRenderCellParams): ChipProps {
+    if (params.value === "Yes") {
+        return {
+            label: params.value,
+            style: {
+                borderColor: green[500],
```

Comment:
I would prefer for these to be defined in the site theme and then accessed using the `useTheme` hook. It will provide better consistency

Reply:
implemented the theme palette